tests/test_minesweeper.py

```diff
@@ -3,55 +3,58 @@
 import numpy as np
 
 from popgym.envs.minesweeper import HiddenSquare, MineSweeper
+from tests.base_env_test import AbstractTest
 
 
-class TestMineSweeper(unittest.TestCase):
-    def test_step(self):
-        e = MineSweeper()
-        e.reset()
-        done = False
-        for i in range(1000):
-            _, _, done, _ = e.step(e.action_space.sample())
-            if done:
-                e.reset()
+class TestMineSweeper(AbstractTest.POPGymTest):
+    def setUp(self) -> None:
+        self.env = MineSweeper()
+
+    # def test_step(self):
```

Comment:
Can we add this test back?

Reply:
This test is already done by the base env test class see `AbstractTest.POPGymTest.test_step`. I will remove the comments.